app/views/docs/oauth-providers/discord.md

```diff
@@ -0,0 +1,350 @@
+
+## Enabling the Discord provider
+Before using Discord to authenticate users, you must enable the provider in your Appwrite console.
+1. Navigate to your Appwrite project
+2. Navigate to **Auth** > **Settings**
+3. Find and open the OAuth provider
+4. In the **Discord OAuth2 Settings** modal, use the toggle to enable the provider
+
+Don't close this modal. We'll need to create a Discord OAuth app to complete this form.
+
+## Creating a Discord OAuth app
+To use Discord OAuth with Appwrite, you must create an OAuth app on your Discord account. You can do this by following the [Discord Developer Portal](https://discord.com/developers/docs/topics/oauth2#oauth2) guide from Discord. 
+
+After you've created your Discord OAuth app, follow these steps:
+- In the *General Information* section, navigate to the **Application ID** and click **Copy**. 
+- Paste this in the **App ID** field in the **Discord OAuth2 Settings** modal from the Appwrite console.
+- Go back to the Discord OAuth app and just under the *Application ID*, copy the **Public Key**.
+- Paste this in the **App Secret** field in the **Discord OAuth2 Settings** modal from the Appwrite console.
+- To complete set up, copy the OAuth2 redirect **URI**.
+- Click the **Update** button to save your changes.
+- Go back to the Discord OAuth app and select the **OAuth2** section in the side navigation bar.
+- In the *General* sub-section, click the **Add Redirect** button.
+- Paste the URI link from the **Discord OAuth2 Settings** modal in your Appwrite console. This will redirect users back to your application after they have authenticated with Discord.
+- Lastly, click the **Save Changes** button.
+
+## Authenticating
+You can use any of the Appwrite Client SDKs to authenticate users with their Discord accounts.
+
+### Web
+When a user calls the [Create OAuth2 Session](https://appwrite.io/docs/client/account#accountCreateOAuth2Session) endpoint in your web app, they will be taken to Discord's OAuth page to complete their login. 
+
+After authenticating, they'll be redirected back to your app using either the `success` or `failure` URLs provided. To provide the best experience to your users, make sure to **implement and provide both routes** to prompt the user about successful and failed authentication attempts.
+
+```js
+import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+    .setProject('[PROJECT_ID]') // Your project ID
```

Comment:
Please align this comment with the above one.

Reply:
Noted

Reply:
Done